pids.md

```diff
@@ -7,6 +7,6 @@ permalink: /pids/
 {% for vid in vids %}
   {% assign urlparts = vid.url|split:"/" %}
   {% assign vidno = urlparts[1] %}
-  <h3><a href="{{vid.url}}">0x{{urlparts[1]}}</a></h3>
+  <h3><a href="{{vid.url}}">0x{{vidno}}</a></h3>
```

Comment:
This line is still generating a pair of p /p around it, which the validator doesn't like. Any thoughts on how to get rid of it @Arachnid ? I wondered if switching the whole line to markdown might work?